src/tunnel.cc

```diff
@@ -186,6 +189,10 @@ class TunnelStateData: public PeerSelectionInitiator
     /// whether another destination may be still attempted if the TCP connection
     /// was unexpectedly closed
     bool retriable;
+
+    /// whether we will not consider using any other server
```

Comment:
Please avoid negative phrasing of a positive-style name.
```suggestion
    /// whether the choice of server is absolute. Indeterminate data/state has likely been shared between it and client.
```

Reply:
I hate negative definitions of booleans, but I could not come up with a better one.

I could not use your suggestion "as is" (for various technical reasons that I can discuss if you insist), but it helped to find a better way to describe this boolean data member (commit 410e396). Thank you! The current code reads: `whether the decision to use a particular destination was final`.

Reply:
looks good. thank you.